prompting/base/validator.py

```diff
@@ -324,16 +328,16 @@ def update_scores(self, rewards: torch.FloatTensor, uids: List[int]):
 
         # Compute forward pass rewards, assumes uids are mutually exclusive.
         # shape: [ metagraph.n ]
-        step_rewards = self.scores.scatter(
+        step_rewards: float = self.scores.scatter(
             0, torch.tensor(uids).to(self.device), rewards.to(self.device)
         ).to(self.device)
         bt.logging.debug(f"Scattered rewards: {rewards}")
 
         # Update scores with rewards produced by this step.
         # shape: [ metagraph.n ]
-        alpha = self.config.neuron.moving_average_alpha
-        self.scores = alpha * step_rewards + (1 - alpha) * self.scores
-        self.scores = (self.scores - self.config.neuron.decay_alpha).clamp(min=0)
+        alpha: float = self.config.neuron.moving_average_alpha
+        self.scores: float = alpha * step_rewards + (1 - alpha) * self.scores
+        self.scores: float = (self.scores - self.config.neuron.decay_alpha).clamp(min=0)
```

Comment:
This type is a tensor of a size of UIDs in the metagraph
```suggestion
        self.scores: torch.Tensor = alpha * step_rewards + (1 - alpha) * self.scores
        self.scores: torch.Tensor = (self.scores - self.config.neuron.decay_alpha).clamp(min=0)
```